docker-compose.yaml

```diff
@@ -0,0 +1,63 @@
+---
+version: '3'
+networks:
+  app:
+services:
+  stn_postgresql:
```

Comment:
I would actually love to have these container names unprefixed. If I remember correctly how `docker-compose` works, one of the advantages of it is that it manages container name uniqueness globally as long as you have unique names within a particular compose setup. That way I can do things like ```docker-compose exec django ./manage.py makemigrations app_name --empty``` without having to remember the prefix for the particular application I'm working on.